### DISCUSSION TITLE
Interpolation of dynamic values Error: INVALID_MESSAGE: Incorrect locale information provided

### DISCUSSION THREAD

**Discussion by artsztu27** [amannn/next-intl#1548] [Q&A]
### Description

I follow the doc. App Router setup without i18n routing adand set my porject.
I followed the [documentation](https://next-intl-docs.vercel.app/docs/getting-started/app-router/without-i18n-routing) steps to set up my project, but I encountered the following issue when trying to use dynamic parameters.

`Error: INVALID_MESSAGE: Incorrect locale information provided
    at getFallbackFromErrorAndNotify (.next/server/chunks/ssr/node_modules_d2c37e._.js:3669:23)
    at translateBaseFn (.next/server/chunks/ssr/node_modules_d2c37e._.js:3735:20)
    at translateFn (.next/server/chunks/ssr/node_modules_d2c37e._.js:3757:24)
    at PasswordInputComponent (/.next/server/chunks/ssr/_e8c17c._.js:26:17) {
  code: 'INVALID_MESSAGE',
  originalMessage: 'Incorrect locale information provided'
}`
**packge.json**
```json

{
  "name": "card-fe-admin",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev --turbopack",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@heroicons/react": "^2.1.5",
    "@radix-ui/themes": "^3.1.5",
    "classnames": "^2.5.1",
    "clsx": "^2.1.1",
    "date-fns": "^4.1.0",
    "lint-staged": "^15.2.10",
    "lodash": "^4.17.21",
    "next": "15.0.2",
    "next-intl": "^3.24.0",
    "react": "19.0.0-rc-02c0e824-20241028",
    "react-dom": "19.0.0-rc-02c0e824-20241028"
  },
  "devDependencies": {
    "@types/lodash": "^4.17.13",
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "eslint": "^8.57.1",
    "eslint-config-next": "15.0.2",
    "husky": "^9.1.6",
    "postcss": "^8",
    "prettier": "^3.3.3",
    "prettier-plugin-tailwindcss": "^0.6.8",
    "tailwindcss": "^3.4.1",
    "typescript": "^5"
  }
}
```
### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on [GitHub](https://github.com/amannn/next-intl/issues) and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

App Router setup without i18n routing
 

### Expected behaviour

` 
"placeholder_with_name": "Please Enter {name}"
const t = useTranslations()
t('placeholder_with_name', { title: 'User Name' })
`

**Comment by amannn** [comment]
Could be this: https://github.com/formatjs/formatjs/discussions/2440.

I'm moving this to a discussion since it's a usage question and no reproduction was provided.
